cookbooks/bcpc-hadoop/attributes/phoenixqs.rb

```diff
@@ -0,0 +1,10 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2
+
+default['bcpc']['hadoop']['phoenix']['phoenixqs']['username'] = 'phoenixqs'
+default['bcpc']['hadoop']['phoenix']['phoenixqs']['principal'] =\
+  "HTTP/#{float_host(node[:fqdn])}@#{node[:bcpc][:hadoop][:kerberos][:realm]}"
+default['bcpc']['hadoop']['phoenix']['phoenixqs']['keytab'] =\
+  "#{node[:bcpc][:hadoop][:kerberos][:keytab][:dir]}/" +
+  node[:bcpc][:hadoop][:kerberos][:data][:spnego][:keytab]
+default['bcpc']['hadoop']['phoenix']['phoenixqs']['localuser'] = true
+default['bcpc']['hadoop']['phoenix']['phoenixqs']['serialization'] = 'JSON'
```

Comment:
Can we add an attribute for ``phoenix.queryserver.http.port`` with value set to the default of ``8765`` so that it can be overwritten.

Reply:
in the environment yuo can add node['bcpc']['hadoop']['hbase']['site_xml'] as an override without having it declared explicitly